client/src/Components/Overview.jsx

```diff
@@ -25,61 +26,49 @@ class Overview extends React.Component{
     this.getReturnPolicy()
   }
   getOverview(){
-    var overviewBulletPoints;
-    return getItemSelection(this.props.randomItemNumber)
-      .then((data) => {
-        overviewBulletPoints = data[0]['overview']
-        console.log(overviewBulletPoints)
-        this.setState({
-          overview: overviewBulletPoints
-        })
-      })
-      .catch((err) => {
-        console.log(err, 'overview failed')
-      })
+    let overviewBulletPoints = [];
+    let overviewLength = false;
+    let overview = this.props.data[0]['overview']
+    console.log(overview)
+    if (overview.length > 0){
+      overviewLength = true;
+      overviewBulletPoints = overview
+    }
+    this.setState({
+      overview: overviewBulletPoints,
+      hasOverview: overviewLength
+    })
   }
 
   getShippingInformation(){
-    var manufacturingTime;
-    return getItemSelection(this.props.randomItemNumber)
```

Comment:
are you doing this now in the parent component and then passing the data down to the Overview component?

Reply:
Yeah I'm doing the get request in the parent component and I'm sending down all of the data into the child